pvlib/tests/test_tracking.py

```diff
@@ -393,6 +393,25 @@ def test_get_irradiance():
 
     assert_frame_equal(irradiance, expected, check_less_precise=2)
 
+    # test with albedo as a Series
+    irrads['albedo'] = [0.5, 0.5]
+    with np.errstate(invalid='ignore'):
+        irradiance = system.get_irradiance(tracker_data['surface_tilt'],
+                                           tracker_data['surface_azimuth'],
+                                           solar_zenith,
+                                           solar_azimuth,
+                                           irrads['dni'],
+                                           irrads['ghi'],
+                                           irrads['dhi'],
+                                           albedo=irrads['albedo'])
+
+    expected = pd.Series(data=[21.05514984, nan], index=times,
+                         name='poa_ground_diffuse')
```

Comment:
I know that we're not that concerned with adding testing for deprecated things, but I would like to understand why adding `albedo=irrads['albedo']` to the `system.get_irradiance` call leads to a Series return type instead of a DataFrame return type.

Reply:
I'm only checking one column of the returned DataFrame.

Reply:
Sorry, not sure how I missed that!